sandbox/dandi.bib

```diff
@@ -0,0 +1,4058 @@
+  doi = {10.48324/DANDI.000019/0.220126.2148},
+  url = {https://dandiarchive.org/dandiset/000019/0.220126.2148},
+  author = {Bouchard, Kristofer E. and Chang, Edward F.},
+  keywords = {electrocorticography (ECoG), speech production},
+  title = {Human ECoG speaking consonant-vowel syllables},
+  publisher = {DANDI Archive},
+  year = {2022}
+}
+
+
+# Take latest as the default
+@misc{dandi.000019,
+  doi = {10.48324/DANDI.000019/0.220126.2148},
+  url = {https://dandiarchive.org/dandiset/000019/0.220126.2148},
+  author = {Bouchard, Kristofer E. and Chang, Edward F.},
+  keywords = {electrocorticography (ECoG), speech production},
+  title = {Human ECoG speaking consonant-vowel syllables},
+  publisher = {DANDI Archive},
+  year = {2022}
+}
+
+
+# DANDISET 000020
+@misc{dandi.000020/0.210913.1639,
+  doi = {10.48324/DANDI.000020/0.210913.1639},
+  url = {https://dandiarchive.org/dandiset/000020/0.210913.1639},
+  author = {Gouwens, Nathan},
+  keywords = {Patch-seq, mouse, visual cortex, interneuron},
+  title = {Patch-seq recordings from mouse visual cortex},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# Take latest as the default
+@misc{dandi.000020,
+  doi = {10.48324/DANDI.000020/0.210913.1639},
+  url = {https://dandiarchive.org/dandiset/000020/0.210913.1639},
+  author = {Gouwens, Nathan},
+  keywords = {Patch-seq, mouse, visual cortex, interneuron},
+  title = {Patch-seq recordings from mouse visual cortex},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# DANDISET 000023
+@misc{dandi.000023/0.210914.1900,
+  doi = {10.48324/DANDI.000023/0.210914.1900},
+  url = {https://dandiarchive.org/dandiset/000023/0.210914.1900},
+  author = {Kalmbach, Brian},
+  keywords = {Patch-seq, human, neocortex, layer 2/3},
+  title = {Patch-seq recordings from human cortex (June 2020)},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# Take latest as the default
+@misc{dandi.000023,
+  doi = {10.48324/DANDI.000023/0.210914.1900},
+  url = {https://dandiarchive.org/dandiset/000023/0.210914.1900},
+  author = {Kalmbach, Brian},
+  keywords = {Patch-seq, human, neocortex, layer 2/3},
+  title = {Patch-seq recordings from human cortex (June 2020)},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# DANDISET 000027
+@misc{dandi.000027/0.210831.2033,
+  doi = {10.48324/DANDI.000027/0.210831.2033},
+  url = {https://dandiarchive.org/dandiset/000027/0.210831.2033},
+  author = {Halchenko, Yaroslav O.},
+  keywords = {development},
+  title = {Test dataset for testing dandi-cli.},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# Take latest as the default
+@misc{dandi.000027,
+  doi = {10.48324/DANDI.000027/0.210831.2033},
+  url = {https://dandiarchive.org/dandiset/000027/0.210831.2033},
+  author = {Halchenko, Yaroslav O.},
+  keywords = {development},
+  title = {Test dataset for testing dandi-cli.},
+  publisher = {DANDI Archive},
+  year = {2021}
+}
+
+
+# DANDISET 000029
+<!DOCTYPE html>
```

Comment:
Ho ho -- we might be identifying problems for dandisets where we have releases and DOI minting has failed! someone might need to look into that!

Reply:
filed 
- https://github.com/dandi/dandi-archive/issues/1953

Meanwhile will adjust script to skip those